undertow/src/main/java/org/wildfly/extension/undertow/UndertowRootDefinition.java

```diff
@@ -87,6 +87,14 @@
                     .setRestartAllServices()
                     .setAllowExpression(true)
                     .setDefaultValue(new ModelNode().set(new ValueExpression("${jboss.node.name}")))
+                    .setAlternatives(Constants.UUID_INSTANCE_ID)
+                    .build();
+    protected static final SimpleAttributeDefinition UUID_INSTANCE_ID =
+            new SimpleAttributeDefinitionBuilder(Constants.UUID_INSTANCE_ID, ModelType.BOOLEAN, true)
+                    .setRestartAllServices()
+                    .setAllowExpression(true)
+                    .setDefaultValue(ModelNode.FALSE)
+                    .setAlternatives(Constants.INSTANCE_ID)
```

Comment:
These 2 instances are not incompatible, thus should not utilize setAlternatives(...).

Reply:
pferraro it came as a request from the feature team because they did not want to have this changed this late at the game. However, as I pointed, now that we're renaming, it makes less and less sense to forbid instance-id definition when obfuscated-instance-id is used.